### DISCUSSION TITLE
Feature: Unique Namespace Per Model

### DISCUSSION THREAD

**Discussion by jgulledge19** [TiagoSilvaPereira/vemto-issues#292] [Ideas & Suggestions]
I am working on a larger project that I would like to be able to put the models into sub folders to match the a related group or category. For example, I have some models like `App\Models\Admin\AdminApprover` and  `App\Models\Accounting\AccountingPaymentRequest`. And there would be several more models in the `Admin` group.

It would be very helpful if I could assign a custom namespace per model and/or group them in a category that would share that namespace. The schema UI could also help in organization of these groups with the shared namespaces/categories by visually having a unique background/header color and also allow to collapse/show as an entire group.

**Comment by ranierif** [comment]
I'm also needing to create models in sub folders. 

For example: `App/Models/Setting/Setting.php`

When will we have this feature?

**Comment by TiagoSilvaPereira** [comment]
@ranierif it will come to Vemto 2, currently under development (no ETA yet)
